Rubberduck.Resources/RubberduckUI.resx

```diff
@@ -1699,4 +1699,46 @@ Update aborted.</value>
 Update aborted.</value>
     <comment>{0} module name, {1} filename</comment>
   </data>
+  <data name="CodeInspections_ComponentType_ActiveXDesigner" xml:space="preserve">
+    <value>ActiveX Designer</value>
+  </data>
+  <data name="CodeInspections_ComponentType_ClassModule" xml:space="preserve">
+    <value>Class Module</value>
+  </data>
+  <data name="CodeInspections_ComponentType_ComComponent" xml:space="preserve">
+    <value>COM component</value>
+  </data>
+  <data name="CodeInspections_ComponentType_DocObject" xml:space="preserve">
+    <value>Document Object</value>
+  </data>
+  <data name="CodeInspections_ComponentType_Document" xml:space="preserve">
+    <value>Document Module</value>
+  </data>
+  <data name="CodeInspections_ComponentType_MDIForm" xml:space="preserve">
+    <value>MDI Form</value>
+  </data>
+  <data name="CodeInspections_ComponentType_PropPage" xml:space="preserve">
+    <value>Property Page</value>
+  </data>
+  <data name="CodeInspections_ComponentType_RelatedDocument" xml:space="preserve">
+    <value>Related Document</value>
+  </data>
+  <data name="CodeInspections_ComponentType_ResFile" xml:space="preserve">
+    <value>Resource File</value>
+  </data>
+  <data name="CodeInspections_ComponentType_StandardModule" xml:space="preserve">
+    <value>Standard Module</value>
+  </data>
+  <data name="CodeInspections_ComponentType_Undefined" xml:space="preserve">
+    <value>Undefined</value>
+  </data>
+  <data name="CodeInspections_ComponentType_UserControl" xml:space="preserve">
+    <value>User Control</value>
+  </data>
+  <data name="CodeInspections_ComponentType_UserForm" xml:space="preserve">
+    <value>UserForm</value>
+  </data>
+  <data name="CodeInspections_ComponentType_VBForm" xml:space="preserve">
```

Comment:
Having `CodeInspections` in the name, tickles.

Reply:
How does this vary from other resources that have identifiers? [DeclarationType_FooBar](https://github.com/rubberduck-vba/Rubberduck/pull/5263#discussion_r342326454) was suggested and it feels like this is similar. Not arguing, legitimately want to know if/how this is different.

Reply:
That's the thing: it isn't. `DeclarationType_*` keys don't encode where the resource keys might be used; they're just... declaration types. This should be the same, they're just component types, it doesn't matter if it's inspections using it, or anything else.

Reply:
The point is that there's an `InspectionsUI.resx` ;)

Reply:
:click: Ah.... Yeah. I see the obvious, now.